server/lib/tuist/qa/agent.ex

```diff
@@ -34,7 +44,7 @@ defmodule Tuist.QA.Agent do
       prompt = """
       You are a QA agent. Test the following: #{prompt}.
 
-      First, understand what you need to test. Then, set up a plan for test the feature and execute on it without asking for additional instructions. Make sure to test **multiple** scenarios. Try to interact with most of the active elements in the feature that should be tested. Take screenshots to analyze whether there are any visual inconsistencies and store the screenshots at /tmp/screenshots.
+      First, understand what you need to test. Then, set up a plan for test the feature and execute on it without asking for additional instructions. Take screenshots to analyze whether there are any visual inconsistencies and store the screenshots at /tmp/screenshots.
```

Comment:
at least for now, I decided to keep the test runs shallower. We can rethink this once we start rolling the feature out.